src/js/controller/instream-adapter.js

```diff
@@ -213,6 +213,7 @@ define([
                     if (_instream === null) {
                         return;
                     }
+                    _instream._adModel.set('state', states.BUFFERING);
```

Comment:
Why not do this after line 210? That way the state is buffering while loading the needed provider.